packages/fluxible-addons-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fluxible-addons-react",
-  "version": "0.2.16",
+  "version": "0.2.17",
```

Comment:
we should bump this to 1.0.0, since using `class` means all apps using fluxible-addons-react need to transpile to ES5 for old browsers.  I hope they do already. But would be good to be on the safe side to bump major here.

Reply:
@pablopalacios we could even make this a 1.0.0-beta.0 version and add in more breaking changes that we need. if you are ok with that, you can remove the version bump from this PR.